src/test/regress/sql/multi_shard_update_delete.sql

```diff
@@ -580,6 +580,11 @@ SET    value_2 = 5 * random()
 FROM   events_test_table
 WHERE  users_test_table.user_id = events_test_table.user_id;
 
+UPDATE users_test_table
```

Comment:
Could use a comment, since this is a rather unusual case in the sense that recursive planning does not take care of it because the query is fully pushdownable, but not allowed because it would lead to inconsistent replicas.

it might also be interesting to add:
```sql
UPDATE users_test_table
SET    value_2 = subquery.random FROM (SELECT user_id, random()
                                       FROM events_test_table) subquery;

UPDATE users_test_table
SET    value_2 = subquery.random FROM (SELECT user_id, random()
                                       FROM events_test_table OFFSET 0) subquery
WHERE  users_test_table.user_id = subquery.user_id;
```